library/proc_macro/src/quote.rs

```diff
@@ -12,7 +12,6 @@ macro_rules! quote_tt {
     ({$($t:tt)*}) => { Group::new(Delimiter::Brace, quote!($($t)*)) };
     (,) => { Punct::new(',', Spacing::Alone) };
     (.) => { Punct::new('.', Spacing::Alone) };
-    (:) => { Punct::new(':', Spacing::Alone) };
```

Comment:
Again, I'm not sure if this should be removed.

Reply:
This change is part of #96149. Let's see what the reviewers of that PR say.